src/BluetoothProvider.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+import { BluetoothNotificationsHookOptions } from "./types/Bluetooth";
+import { BluetoothContext } from "./BluetoothContext";
+import { useBluetoothNotifications } from "./useBluetoothNotifications";
+
+export const BluetoothProvider: React.FC<React.PropsWithChildren<BluetoothNotificationsHookOptions>> = ({
```

Comment:
minor but `React.FC` already extends the provided type with `children` so the extra `React.PropsWithChildren` generic type should not be needed

The following should be enough
```
export const BluetoothProvider: React.FC<BluetoothNotificationsHookOptions> = ({
```

Reply:
That's very helpful to know! Thanks!